conbench/entities/benchmark_result.py

```diff
@@ -486,23 +486,30 @@ def ui_mean_and_uncertainty(self) -> str:
 
     # maybe make this a cached property
     @property
-    def ui_rel_sem(self) -> str:
+    def ui_rel_sem(self) -> Tuple[str, str]:
+        """
+        The first string is a stringified float for sorting in the table.
+        The second string is for display in the table, with unit.
+        Associate "n/a" with a negative value so that DESC sorting by error
+        is helpful.
```

Comment:
Will need to make this interface better in the (near) future. This works for now, moving on.